recipes/classix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: classix
+  version: 0.1.1
+
+
+source:
+  fn: ClassixClustering-0.5.8.tar.gz
+  url: https://files.pythonhosted.org/packages/6d/24/b1610d481857cf97815e34e366d7c384e4910a07b73510b7d952926699b6/ClassixClustering-0.5.8.tar.gz
+  sha256: 9a4b03d4aa1e47ccc9e92fb7152c9c3c417a7e60867c7df33214e6487f4e2a9a
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+    
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - setuptools
+    - pandas
+    - numpy 
+    - cython
+    - scipy ==1.7.2
+    - requests
+    - matplotlib-base
+
+  run:
+    - python >=3.6
+    - pandas
+    - requests
+    - numpy
+    - cython
```

Comment:
You can likely remove `cython` as a run time dependency.

Reply:
Thanks, modified!